pkg/cmd/root/help.go

```diff
@@ -16,7 +18,14 @@ import (
 func rootUsageFunc(w io.Writer, command *cobra.Command) error {
 	fmt.Fprintf(w, "Usage:  %s", command.UseLine())
 
-	subcommands := command.Commands()
+	var subcommands []*cobra.Command
+	for _, c := range command.Commands() {
+		if !c.IsAvailableCommand() {
```

Comment:
We still want to show usage for commands that have only nested alias sub commands.